samples/CosmosDbIntegtation/ReadThroughSamples/ListSample.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Microsoft.Azure.Cosmos;
+using Microsoft.Azure.Cosmos.Linq;
+using Microsoft.Azure.Functions.Worker.Extensions.Redis;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using StackExchange.Redis;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Policy;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Redis.Samples
+{
+    public static class ListSample
+    {
+        //Redis Cache primary connection string from local.settings.json
+        public const string redisConnectionString = "redisConnectionString";
+        private static readonly IDatabase s_redisDb = ConnectionMultiplexer.Connect(Environment.GetEnvironmentVariable(redisConnectionString)).GetDatabase();
+
+        //CosmosDB database name and container name from local.settings.json
+        public const string CosmosDbDatabaseId = "CosmosDbDatabaseId";
+        public const string CosmosDbContainerId = "CosmosDbContainerId";
+
+        /// <summary>
+        /// Adds a value to a Redis cache.
+        /// </summary>
+        /// <param name="response"> The response object returned by a Cosmos DB query. </param>
+        /// <param name="item"> The item to be added to the Redis cache. </param>
+        /// <param name="listEntry">The key for the Redis list to which the item will be added. </param>
+        /// <returns> None </returns>
+        public static async Task ToCacheAsync(FeedResponse<CosmosDBListData> response, CosmosDBListData item, string listEntry)
+        {
+            //Retrieve the values in cosmos associated with the list name, so you can access each item
+            var fullEntry = response.Take(response.Count);
+
+            if (fullEntry == null) return;
+
+            //Accessing each value from the entry 
+            foreach (CosmosDBListData inputValues in fullEntry)
+            {
+                RedisValue[] redisValues = Array.ConvertAll(inputValues.value.ToArray(), item => (RedisValue)item);
+                await s_redisDb.ListRightPushAsync(listEntry, redisValues);
+
+                //Optional foreach loop + console write line to confirm each value is sent to the cache
```

Comment:
This is more of a DEBUG functionality. I'd recommend removing it or adding it to a logger if you think it's relevant.